sympy/sets/sets.py

```diff
@@ -67,13 +67,13 @@ class Set(Basic, EvalfMixin):
     is_Interval = False
     is_ProductSet = False
     is_Union = False
-    is_Intersection = None  # type: Optional[bool]
-    is_UniversalSet = None  # type: Optional[bool]
-    is_Complement = None  # type: Optional[bool]
+    is_Intersection: FuzzyBool = None
+    is_UniversalSet: FuzzyBool = None
+    is_Complement: FuzzyBool = None
```

Comment:
I think these should really be `bool`. Probably they should just be set to False here.